myob/endpoints.py

```diff
@@ -71,6 +71,9 @@
         'name': 'inventory',
         'methods': [
             (CRUD, 'Item/', 'inventory item'),
+            (ALL, 'ItemPriceMatrix/', 'inventory item price matrice'),
```

Comment:
Ha, could've probs added a line to the pluralisation code instead. :P
https://github.com/uptick/pymyob/blob/03b87b9a7a6ced5bafc04630002e506449ace2b2/myob/utils.py#L1-L5